### ISSUE TITLE
(Bug): Like button number showing up weird whenever i click like button

### ISSUE THREAD

**Issue by TwistyNuts** [Anarios/return-youtube-dislike#604]
### Browser

Brave Browser

### Browser Version

Version 1.38.115 Chromium: 101.0.4951.64 (Official Build) (64-bit)

### Extension or Userscript?

Extension

### Extension/Userscript Version

v3.0.0.1

### Video link where you see the problem

any video with alot of likes

### What happened?

<img width="212" alt="لقطة الشاشة 2022-05-11 160245" src="https://user-images.githubusercontent.com/105236450/167947747-10ad51a8-52a4-438a-a6e6-88ba516178e9.png">
This shows up when i select 123 thousand number format

<img width="210" alt="لقطة الشاشة 2022-05-11 160427" src="https://user-images.githubusercontent.com/105236450/167947755-9da2190c-5008-4d24-a3b1-c233e365bb91.png">
This shows up when i select 123K number format

(it stays until i refresh the page)

### How to reproduce/recreate?

Basically just like a video with alot of likes

**Comment by PickleNik** [comment]
Known issue, it's duplicating, disable Like Re-Formatting in settings for now before a fix is pushed in an update.

**Comment by TwistyNuts** [comment]
alright, thank you (hope it gets fixed soon)
